packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -362,7 +362,7 @@ class StrutStyle extends Diagnosticable {
   /// Equivalent to having no strut at all. All lines will be laid out according to
   /// the properties defined in [TextStyle].
   static const StrutStyle disabled = StrutStyle(
-    height: 0.0,
+    height: 1.0,
```

Comment:
`null` may be the better choice here, as 1.0 is indicating it should be exactly the fontSize height, ignoring the font provided height

Reply:
Actually, 0.0 is correct, since Strut is meant as a minimum line height. Disabled strut with 1.0 would prevent text from being smaller than the fontSize, which would defeat the purpose of a "disabled" strut, which is meant to have the effect of there being no strut on the paragraph.

Reply:
The 1.0 strut would probably be better under a separately named static constant, maybe `regular` or something.